WaveInt.py

```diff
@@ -378,7 +379,7 @@ def synthSDR(self,out_type,strike,dip,rake,M0,stf_type,duration=None,rfile=None,
             self.synth[stat]={}
             for c in 'ZNE':
                 sacfile = sac()
-                sacfile.read('%s_%c.SAC'%(stat,c))
+                sacfile.rsac('%s_%c.SAC'%(stat,c))
```

Comment:
Actually, in the last version of sacpy, the method `rsac` has been renamed `read`. Please remove that change and update sacpy (cf., https://github.com/zduputel/sacpy)

Reply:
It seems that waveInt is still importing sacpy from the local copy of the package, which doesn't include read. Maybe  change the importing sacpy line.

Reply:
Please update your local version of the package (WaveMod does not inclure a local copy of sacpy). If you are using the w-phase package, please make sure that the $WPhase package is not in your PYTHONPATH.

Reply:
Good point, I have removed the local copy of sacpy in the master branch. Please use the sacpy module here: https://github.com/zduputel/sacpy